.gitlab-ci.yml

```diff
@@ -334,7 +357,7 @@ osx-cuda:
 clang:6.0:
   <<: *global_job_definition
   stage: build
-  image: gitlab.icp.uni-stuttgart.de:4567/espressomd/docker/clang-python3:6.0
+  image: docker.pkg.github.com/espressomd/docker/clang:446ff604bbfa63f30ddb462697fa0d0dc2630460
   script:
     - export myconfig=maxset with_coverage=false with_static_analysis=true with_asan=true with_ubsan=true test_timeout=900
```

Comment:
it would probably a good idea to get rid of all those ```export ...``` stuff and use the gitlab ci variables instead. This makes it a lot more readable.

Reply:
agreed, but I've already altered all these export lines in #3582, so if we do this change now, I'll have to do it again in #3582 to resolve the merge conflicts